File: app/src/main/java/com/example/focusstartsecondpart/features/guests/data/network/GuestsLoader.java
package com.example.focusstartsecondpart.features.guests.data.network;

import com.example.focusstartsecondpart.features.guests.domain.model.Guest;
import com.example.focusstartsecondpart.features.guests.domain.model.Result;
import com.example.focusstartsecondpart.features.guests.domain.model.VerifiedMember;

import java.util.List;

import io.reactivex.Single;

public interface GuestsLoader {
    Single<List<Guest>> loadGuests(int id);
    Single<Result> updateGuest(int id, List<VerifiedMember> verifiedMemberList);
}

File: app/src/main/java/com/example/focusstartsecondpart/features/guests/data/network/GuestsLoader.kt
package com.example.focusstartsecondpart.features.guests.data.network

import com.example.focusstartsecondpart.features.guests.domain.model.Guest
import com.example.focusstartsecondpart.features.guests.domain.model.Result
import com.example.focusstartsecondpart.features.guests.domain.model.VerifiedMember

import io.reactivex.Single

interface GuestsLoader {
    fun loadGuests(id: Int): Single<List<Guest>>
    fun updateGuest(eventId: Int, verifiedMemberList: List<VerifiedMember>): Single<Result>
}
